blog/scss/component/_button.scss

```diff
@@ -0,0 +1,25 @@
+.oringeBtn {
+  border: 2px #ff9671 solid;
+  padding: 0.9em 2em;
+  background: white;
+  color: #ff9671;
+  font-size: 1rem;
+  cursor: pointer;
+  border-radius: 5px;
+
+  &:hover {
+    background-color: rgba(255, 94, 87, 0.4);
+    transition: 1s;
+  }
+
+  &-L {
+    width: 11rem;
+    background-color: rgb(255, 94, 87);
+    font-size: 100%;
+    color: white;
+
+    &:hover {
+      background-color: rgba(255, 94, 87, 0.5);
+    }
+  }
+}
```

Comment:
Please use the button component from shared folder, and also you can get padding, font-size, border-radius .. . values from shared variables too :)